projects/kubernetes/kube-master.yml

```diff
@@ -51,18 +51,19 @@ services:
      - /dev:/dev
      - /etc/resolv.conf:/etc/resolv.conf
      - /lib/modules:/lib/modules
+     - /run:/run:rshared,rbind
```

Comment:
Ah, I got it, on the control plane side things are mounted/populated under `/var/lib/kubernetes/...` they are then bound into various runtime containers as `/var/run/secrets` which is why I was seeing that. But the runtime mount is a kube managed mount not something we need to be concerned with and I think there is indeed no need for `/run` to be shared by the control plane and the host.

I'll double check by actually making the change rather than just inspecting things, but expect an update.